frappe/core/doctype/user/user.py

```diff
@@ -983,4 +983,19 @@ def throttle_user_creation():
 	if frappe.flags.in_import:
 		return
 	if frappe.db.get_creation_count('User', 60) > 60:
-		frappe.throw(_('Throttled'))
\ No newline at end of file
+		frappe.throw(_('Throttled'))
+
+@frappe.whitelist()
+def update_roles(role_name):
+	users = frappe.get_all('User', filters={'role_name': role_name})
```

Comment:
should be just `role_profile`

https://github.com/frappe/erpnext/wiki/Naming-Guidelines